validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkSubmissionValidatorTest.java

```diff
@@ -131,43 +142,40 @@ public class BulkSubmissionValidatorTest extends AbstractValidatorUnitTest {
     /**
      * The number of requests.
      */
-    private final long numberOfRequests = 1L;
+    private static final long NUMBER_OF_REQUESTS = 1L;
 
     /**
      * Request id.
      */
 
-    private final long requestId = 20L;
+    private static final long REQUEST_ID = 20L;
 
     /**
      * Data retention period.
      */
-    private final int dataRetentionPeriod = 90;
+    private static final int DATA_RETENTION_PERIOD = 90;
 
     /**
      * SDT Bulk reference.
      */
-    private String sdtBulkReference = "sdtBulkReference";
+    private static final String SDT_BULK_REFERENCE = "sdtBulkReference";
 
     /**
      * Current date time.
      */
-    private LocalDateTime now = LocalDateTime.now();
+    private static final LocalDateTime NOW = LocalDateTime.now();
 
     /**
      * Setup of the Validator and Domain class instance.
      */
+    @BeforeEach
+    @Override
     public void setUpLocalTests() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
I think this isn't needed because the @ExtendWith(MockitoExtension.class) annotation initialises the mock objects.